knowledge-base/error-promise-is-undefined.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Error is thrown on browsers not supporting promises.
```

Comment:
Always use the error in the title when it is known. Also, the title should have an action. For example: "'Promise' is undefined error on loading the Report Viewer in the browser"